src/Matmar10/Money/Entity/Money.php

```diff
@@ -79,17 +112,17 @@ public function getCurrency()
 
     public function getScale()
     {
-        return $this->scale;
+        return $this->scale ?: $this->scale = bcpow(10, $this->currency->getPrecision(), 0);
```

Comment:
What's the goal of moving the assignment here? Now their is a condition being evaluated any time someone calls getScale()